src/styles/globals.css

```diff
@@ -41,7 +41,21 @@ button {
 }
 
 :root {
-  --header-height: 52px;
+  --header-inner-height: 52px;
+}
+
+:root {
+  --header-height: var(--header-inner-height);
+}
+
+.psaBanner:root {
+  --header-height: calc(var(--header-inner-height) * 2);
+}
+
+@media (max-width: 600px) {
+  .psaBanner:root {
+    --header-height: calc(var(--header-inner-height) * 2 + 16px);
```

Comment:
This seems to be very specific to the current banner that takes up 2 lines on smaller screens but it would not work for longer content. I would suggest adding a comment here to explain that.

We could also keep the same height but make the inner content scrollable to cover smaller screens.

Reply:
Actually on my phone it takes 3 lines so this magic number won’t work. Although the worst that happens is it’s still partially covering some content.  
I don’t like the scroll idea tbh.

Reply:
Added a comment.